src/python/gem5/simulate/exit_event_generators.py

```diff
@@ -103,3 +104,33 @@ def default_workend_generator():
     while True:
         m5.stats.dump()
         yield False
+
+
+def default_simpoint_generator():
+    """
+    A default generator for SimPoints. It will do nothing.
+    """
+    defaultBehaviorWarning(
+        "default_simpoint_generator",
+        "A default generator for SimPoints. It will do nothing.")
+    while True:
+        yield False
+
+def simpoint_dump_reset_generator():
```

Comment:
Does this have to be "simpoint" specific, or is this just a dump/reset generator?